src/Cache.php

```diff
@@ -104,15 +123,16 @@ public function has($key): bool
      */
     public function getMultiple($keys, $default = null): iterable
     {
+        // TODO refactor
         $keyMap = [];
         foreach ($keys as $key) {
-            $keyMap[$key] = $this->buildKey($key);
+            $keyMap[$key] = $this->normalizeKey($key);
         }
-        $values = $this->handler->getMultiple(array_values($keyMap));
+        $values = $this->handler->getMultiple(array_values($keyMap), $default);
         $results = [];
         foreach ($keyMap as $key => $newKey) {
             $results[$key] = $default;
-            if (isset($values[$newKey])) {
+            if (array_key_exists($newKey, (array)$values)) {
```

Comment:
`getMultiple` can not return non-array.

Reply:
It returns `iterable` I'll use `iterableToArray` here like in other places.